internal/interpreter/api.rs

```diff
@@ -1215,15 +1215,7 @@ impl ComponentInstance {
         }
     }
 
-    /// Highlight the elements which are pointed by a given source location.
-    ///
-    /// WARNING: this is not part of the public API
-    #[cfg(feature = "highlight")]
-    pub fn highlight(&self, path: PathBuf, offset: u32) {
-        crate::highlight::highlight(&self.inner, path, offset);
-    }
-
-    /// Highlight the elements which are pointed by a given source location.
+    /// Highlight the components which are pointed by a given source location.
```

Comment:
this doesn't highlight anything. IT returns the positions   (same for the other function)
Also, I don't understand why we need a different function for component and element.  you can just ask for the position of the component.root_element